### PR TITLE
Feature/oc 181 profile page setup

### PR DESCRIPTION
**Pull request by DrKlonk**
Creates a page for users/<username>
Remove the author page
Change the visuals and links on Article pages and CommentSection

No design is implemented yet, this is done in https://one-community.atlassian.net/browse/OC-184

### CODE REVIEW

**Comment by dennis-ordina** [comment]
Er zit nog een build error in:

Screenshot 2022-03-30 at 16 52 54

File: components/global/CommentItem.vue [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <template>
   <div class="comment">
-    <div class="comment__user">{{ user }}</div>
+    <div class="comment__user">
+      <NuxtLink class="comment__user-link" :to="`/users/${username}`">{{
```
Comment (dennis-ordina) [inline components/global/CommentItem.vue:4]: Nice

File: store/users.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+export type RootState = ReturnType<typeof state>
+
+export const getters: GetterTree<RootState, RootState> = {
+  users: (state) => state.users,
+}
+
+export const mutations: MutationTree<RootState> = {
+  ADD_USER: (state, newUser: User) => {
+    console.log('pushing', newUser)
```
Comment (dennis-ordina) [inline store/users.ts:19]: Can you remove this?

**Review by dennis-ordina — CHANGES_REQUESTED** [review]

**Comment by dennis-ordina** [comment]
Can you change the seed-db script to reflect the new structure as well?

File: components/global/CategoryCard.vue [hunk 3]
```diff
@@ -120,7 +120,7 @@ export default defineComponent({
 })
 </script>
 
-<style lang='scss' scoped>
+<style lang="scss" scoped>
```
Comment (dennis-ordina) [inline components/global/CategoryCard.vue:123]: Hier ook weer double quotes xd weird

**Review by dennis-ordina — APPROVED** [review]

File: components/global/CategoryCard.vue (hunk 3 above)
Comment (DrKlonk) [reply components/global/CategoryCard.vue:123]: Hier is t in de html, dus zoals verwacht

**Pull request merged**